app/services/woms.js

```diff
@@ -60,7 +60,7 @@ export default Service.extend({
 
   socketMessageHandler(event) {
     let data = JSON.parse(event.data);
-    if (data.Item && data.Item.metadata) {
+    if (data) {
```

Comment:
this looks familiar -- does this clobber the workaround you put in the other day? should be fixed by rebase, right?

Reply:
Good catch!